lib/pdffield.js

```diff
@@ -165,7 +165,11 @@ let PDFField = (function PDFFieldClosure() {
             }
         }, _getFieldBaseData.call(this, box));
 
-        this.Boxsets.push({boxes:[anData]});
+        if(box.fieldValue) {
+            anData.checked = box.fieldValue !== 'Off';
```

Comment:
I don't know enough about PDFs to know the answer to this... is it always with this casing?